packages/ticker/src/Ticker.ts

```diff
@@ -419,6 +439,11 @@ export class Ticker
             // during the emit, we can still check for head.next
             const head = this._head;
 
+            if (!head)
+            {
+                return;
+            }
```

Comment:
Maybe it make sense to  check this.head, and only after create a variable ?

```suggestion
            if (!this._head)
            {
                return;
            }
            const head = this._head;
```